settings/default/map.lua

```diff
@@ -105,14 +105,14 @@ xi.settings.map =
     -- Enable/disable jobs other than BST and RNG having widescan
     ALL_JOBS_WIDESCAN = true,
 
-    -- Modifier to apply to player speed. 0 is the retail accurate default. Negative numbers will reduce it.
+    -- Modifier to apply to player speed. 0 is the retail accurate default. Negative numbers will reduce it. This affects all entity base speed as well as max speed.
```

Comment:
your addition to the comment technically true but we overwrite that almost instantly, so in the end this will not influence the final result of objects that have their own separate speed settings here, and I think also will fail to influence pets and trusts. I left some commenting about how bizarrely set up that was in the core code, that everything "begins" with the same base entity speed and is assumed a player until the sql loads happen for mobs and and NPCs and then poof that number is gone. As much as I wanted that to not be the case, I don't think Xaver's speed rewrite tackled that aspect and if it had, this would then be fully not true.

tldr this is just going to confuse people about something that isn't supposed to happen.

Reply:
oh interesting, good point. I wasn't thinkg that yea mob speeds are defined in the db explicitly